src/components/SectionEditModal/index.js

```diff
@@ -0,0 +1,3 @@
+import SectionEditModal from './SectionEditModal'
+
+export default SectionEditModal
```

Comment:
With @trollepierre we will create the comity against the index file!

Reply:
Kind of a pity for guys working in digital